showcase.json

```diff
@@ -1958,5 +1958,46 @@
     "twitter": "",
     "youtube": "",
     "submittableId": "34425295"
+  },
+  {
+    "name": "ETH Integration (Motoko)",
+    "description": "The integration of ethereum on the IC network without bridges is achieved through chain-key (threshold signature) technology for ECDSA signatures, and the smart contracts of IC can directly access the RPC nodes of ethereum through HTTPS Outcall technology. ",
+    "website": "https://avjzx-pyaaa-aaaaj-aadmq-cai.raw.ic0.app/loginByExists",
+    "logo": "/img/showcase/eth-integration-motoko_logo.webp",
+    "screenshots": [],
+    "video": "",
+    "display": "Normal",
+    "id": "eth-integration-motoko",
+    "oneLiner": "uses the Motoko to implement the integration of ckETH.",
+    "stats": "It is being tested",
+    "tags": [
+      "DeFi"
+    ],
+    "usesInternetIdentity": false,
+    "github": "",
+    "twitter": "",
+    "youtube": "",
+    "submittableId": "36483996"
+  },
+  {
+    "name": "BTC integration in ICDex",
```

Comment:
ICDex is already in the showcase and this tile seems somewhat empty. In this state I don't think it should be added

Reply:
Hello, what we added was not the ICDex project, but projects related to BTC and ETH integration. These projects are primarily intended for grant applications.

Reply:
Hello @elephant-w We allow only one listing per core product on the showcase. Would you be open to highlighting that you offer a BTC and ETH integration on the ICLighthouse listing instead?